### DISCUSSION TITLE
Anyone willing/able to sell the hardware ready for usage

### DISCUSSION THREAD

**Discussion by foobar270** [BenediktSeidl/prpd#2] [General]
Hi,

this project looks very promising... however I lack the soldering skills etc. 
Is there any chance of buying ready-to-use module that I can connect to a raspberry PI?

Many thanks for any volunteers.
M.

**Comment by BenediktSeidl** [comment]
If soldering is the only problem: You don't really need it: just cut a short patch cable and search for the following wires:

| wire number | color (wire number is more important!)  | RS485 function  |
| ------------- |------------|------|
| 3      | white/green | A- |
| 6      | green     |  B+ |
| 1 | white/orange      | GND |

*there are two different standards for the colors of a patch cable, so be sure to check with the numbers!*

you can use a cat6 coupling to plug you cable between the powerrouter and the sensor and connect the open lines to a rs485 receiver.

you may use a usb rs485 dongle and connect the wires to the dongle. I'm currently using a similar setup as I have not finish the hardware yet, but have some problems with the rs485 usb dongle, it does not receive any data after some hours. Haven't time to debug this yet, I hope it's a problem of the cheap dongle...
